content/post/21-12-21-tt-IvI/index.md

```diff
@@ -1,58 +0,0 @@
----
```

Comment:
was there a good reason to delete these posts? Of course, the searches have been completed, but maybe it's OK to keep it around as "old news items"?